src/mono/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -110,6 +110,7 @@
     <FeaturePortableTimer Condition="'$(TargetsBrowser)' != 'true'">true</FeaturePortableTimer>
     <FeaturePortableThreadPool Condition="'$(TargetsBrowser)' != 'true'">true</FeaturePortableThreadPool>
     <FeaturePerfTracing Condition="'$(TargetsBrowser)' != 'true'">true</FeaturePerfTracing>
+    <FeatureManagedThreading>true</FeatureManagedThreading>
```

Comment:
I do not think this deserves own Feature flag. The two places where it is used can just use `'$(FeatureCoreCLR)' != 'true'`. It won't be the first condition like that in System.Private.CoreLib.Shared.projitems .